site/en/docs/extensions/mv3/getstarted/index.md

```diff
@@ -58,8 +57,11 @@ manifest, a generic icon will be created for the extension.
 The extension is now installed, but it doesn't currently do anything because we haven't told it what
 to do or when to do it. Let's fix that by adding some code to store a background color value.
 
-To do this, we will need to create a [background script][1] and add it to the extension's manifest.
-Start by creating a file named `background.js` inside the extension's directory.
+### Register the background script in the manifest {: #background_manifest }
```

Comment:
We haven't been real consistent about this, but I think we prefer #background-manifest and not #background_manifest.

Reply:
Good one! I'll use this format from now on :)